app/src/errors/connecionError.js

```diff
@@ -0,0 +1,19 @@
+class ConnectionError extends Error {
+    
+    constructor(e) {
+        // Pasa los argumentos restantes (incluidos los específicos del proveedor) al constructor padre
+        super(e)
+    
+        // Mantiene un seguimiento adecuado de la pila para el lugar donde se lanzó nuestro error (solo disponible en V8)
+        if (Error.captureStackTrace) {
+          Error.captureStackTrace(this, ConnectionError)
+        }
+    
+        this.name = 'ConnectionError'
+        // Información de depuración personalizada
+        this.message = 'Error connecting to Python service'
```

Comment:
Idem arriba...quizás '...to server'
Más que nada porque los errores son generales y se podrían aplicar a otro server sin problema. El que hace el llamado es el que sabe a quien le esta pegando, la clase error no

Reply:
idem sugerencia del constructor o simil del auth para hacer el mensaje más descriptivo